src/test/resources/fixtures/hmh/hmh-mi/development/caliperEventGradeGraded.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id":"urn:uuid:a50ca17f-5971-47bb-8fca-4e6e6879001d",
+  "type": "GradeEvent",
+  "actor":{
+    "id":"urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+    "type":"Person"
+  },
+  "action": "Graded",
+  "object": {
+    "id": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f",
+    "type": "Attempt",
+    "assignee": "urn:uuid:0f4fedbe-2227-415f-8553-40731a627171",
+    "assignable": {
+      "id":"urn:uuid:c050e852-5edb-4743-92d1-b53466de3a5f",
+      "type": "Assessment"
+    },
+    "count": 35,
+    "dateCreated": "2016-11-15T10:05:00.000Z",
+    "startedAtTime": "2016-11-15T10:05:00.000Z",
+    "endedAtTime": "2016-11-15T10:55:12.000Z"
+  },
+  "eventTime": "2016-11-15T10:57:06.000Z",
+  "edApp": "https://www.hmhco.com/MathInventory",
+  "generated": {
+    "id": "urn:uuid:2050e852-5edb-4743-92d1-b53466de3a5f",
+    "type": "Score",
+    "attempt": "urn:uuid:1050e852-5edb-4743-92d1-b53466de3a5f",
```

Comment:
Observation: attempt id here is nearly identical to generated.id - these are definitely different ids and see comment above for reference/help with Caliper fields  https://github.com/hmhco/caliper-common-fixtures/pull/10/files#diff-060d5b569924f23c66224a63ba8a5ddbR15